packages/react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInstructionSetExternalRuntime.js

```diff
@@ -138,4 +134,9 @@ export function completeBoundaryWithStyles(
   );
 }
 
+function cleanupWith(cb) {
```

Comment:
You can't declare helpers like this because they end up being globals. Do it inline instead.